Microsoft.Toolkit.Uwp.UI/Converters/DateTimeToStringConverter.cs

```diff
@@ -0,0 +1,62 @@
+﻿// ******************************************************************
```

Comment:
I'm not sure I see the point of this converter. Why not just do the formatting in the viewmodel, that's what it's for.